scalameta/parsersJSFacade/src/main/scala/scala/meta/parsersjsfacade/Api.scala

```diff
@@ -0,0 +1,59 @@
+package scala.meta
+package parsersjsfacade
+
+import scala.scalajs.js
+import js.JSConverters._
+import js.annotation._
+
+import parsers._
+
+object ScalaParser {
```

Comment:
on a second thought, it shouldn't make a difference, since`@JSExport` is not used anywhere else in scalameta. The main difference is whether the export becomes a global variable or it is exported with the commonjs format, but again this shouldn't break anything.